packages/otel/src/vercel-request-context/exporter.ts

```diff
@@ -27,10 +28,15 @@ export class VercelRuntimeSpanExporter implements SpanExporter {
       //   // TODO Fix this
       //   useLongBits: false,
       // }
-      const data = createExportTraceServiceRequest(spans, {
-        useHex: true,
-        useLongBits: false,
-      });
+      const serializedData = JsonTraceSerializer.serializeRequest(spans);
```

Comment:
It looks like `useLongBits` comment above is no longer relevant. Could you drop it?